groups/sig-autoscaling/groups.yaml

```diff
@@ -79,14 +83,14 @@ groups:
       ReconcileMembers: "true"
     members:
       - bskiba@google.com
-      - maciekpytel@google.com
-      - ksiedlecki@google.com
-      - jbartosik@google.com
-      - jtuznik@google.com
+      - bwroblewski@google.com
       - danielmk@google.com
       - guyjtempleton@googlemail.com
+      - jack.francis@microsoft.com
+      - jbartosik@google.com
```

Comment:
We can remove these:
* jbartosik@google.com
* kgolab@google.com

AFAIK, the rest of google.com addresses here are people still working on Cluster Autoscaler or VPA at least occasionally.

@raywainman to confirm the VPA side.

Reply:
confirmed for VPA folks